services-worker/src/main/java/edu/unc/lib/dl/cdr/services/worker/WorkerDaemon.java

```diff
@@ -38,18 +40,22 @@ public void start() throws Exception {
 			appContext.refresh();
 		}
 		
-		WorkerPool workerPool = appContext.getBean(WorkerPool.class);
-		workerPool.getWorkerEventEmitter().addListener(this);
-		workerPool.run();
+		Map<String, WorkerPool> workerPools = appContext.getBeansOfType(WorkerPool.class);
+		for (WorkerPool workerPool : workerPools.values()) {
+			workerPool.getWorkerEventEmitter().addListener(this);
+		}
+		workerPools.values().iterator().next().run();
```

Comment:
This is weird, but calling run() on each workerPool resulted in it calling start on threads which had already started in the second worker pool.  There are no messages for workers in the second pool starting, but they still do work